go.mod

```diff
@@ -27,6 +27,10 @@ require (
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.1.0
 	github.com/k8snetworkplumbingwg/sriov-cni v2.1.0+incompatible
 	github.com/kevinburke/ssh_config v0.0.0-20190725054713-01f96b0aa0cd
+	github.com/mdlayher/arp v0.0.0-20191213142603-f72070a231fc
+	github.com/mdlayher/ethernet v0.0.0-20190606142754-0394541c37b7
+	github.com/mdlayher/ndp v0.0.0-20210831201139-f982b8766fb5
+	github.com/mdlayher/raw v0.0.0-20211126142749-4eae47f3d54b
```

Comment:
Why do we import those packages as we have implemented some functions in agent/util/arping and  agent/util/arping?

Reply:
This is feature is not just for sending GARP or NS over the device. We need to listen for ARP and NS requests and process the requests accordingly. With out these packages we may need to manage the raw sockets, parse the ethernet headers, etc.